src/app/common/components/presentation/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,17 @@
+<mat-card>
+  <mat-card-title>
+    SidenavBasic
```

Comment:
is this being used?

Reply:
No, I wasn't sure if I should add the sidebar files in case we use them later, but I can get rid of them.